includes/class-sensei-settings.php

```diff
@@ -15,6 +15,8 @@
  */
 class Sensei_Settings extends Sensei_Settings_API {
 
+	const VISITED_SECTIONS_OPTION_KEY = 'sensei_settings_sections_visited';
```

Comment:
Given that we're adding the key as a const in `Sensei_Settings`, couldn't we rely on `Sensei()->settings->get()` and `Sensei()->settings->set()` too? I'm not sure. :thinking:

Reply:
Honestly... this is a really good point. I like the idea and if it isn't any problem to have a list inside those settings I will try to implement it that way.

Reply:
I gave it a thought and finally I will not changing this. I saw that these `Sensei()->settings` are more user oriented and I want to keep this "visited" sections more of a hidden value.